openlr/openlr_match_quality/openlr_assessment_tool/traffic_mode.hpp

```diff
@@ -78,12 +179,40 @@ class TrafficMode : public QAbstractTableModel
 
   Qt::ItemFlags flags(QModelIndex const & index) const Q_DECL_OVERRIDE;
 
+  bool IsBuildingPath() const { return m_buildingPath; }
+  void StartBuildingPath();
+  void PushPoint(m2::PointD const & coord,
+                 std::vector<FeaturePoint> const & points);
+  void PopPoint();
+  void CommitPath();
+  void RollBackPath();
+
+  size_t GetPointsCount() const;
+  m2::PointD const & GetPoint(size_t const index) const;
+  m2::PointD const & GetLastPoint() const;
+  std::vector<m2::PointD> GetCoordinates() const;
```

Comment:
Path is somewhat more complex thing than a vector of points. I can agree on GetPoints(), and it would be consistent with one in decoded_path.hpp. What do you think?